Assignment2/target/classes/views/AcesUp/AcesUp.flt.html

```diff
@@ -153,10 +218,11 @@
 contentType:"application/json; charset=utf-8",
 dataType:"json",
 });
-
+document.getElementById("score").innerHTML = "Score: " + game.Score;
```

Comment:
Shouldn't this line be in the success part of the ajax call. Do you want to update the game's score if the post fails?